mockwebserver/src/main/java/okhttp3/mockwebserver/MockResponse.java

```diff
@@ -254,6 +257,21 @@ public long getBodyDelay(TimeUnit unit) {
   }
 
   /**
+   * Set the time to {@code delay} prior to shutting down an HTTP/2 connection. This is only valid
+   * with {@link SocketPolicy#GRACEFUL_SHUTDOWN_AT_END}. Use this to simulate a graceful HTTP/2
+   * connection shutdown.
+   */
+  public MockResponse setShutdownDelay(long delay, TimeUnit unit) {
+    shutdownDelay = delay;
+    shutdownDelayUnit = unit;
+    return this;
+  }
+
+  public long getShutdownDelay(TimeUnit unit) {
```

Comment:
I like the idea of storing the milliseconds. I do not have a good reason for doing it this way other than that's how the body delay works. Any strong preference?